pyExample_alg01/pyExample_alg01.py

```diff
@@ -0,0 +1,5 @@
+import rounder
+
+r = rounder.Rounder()
+num = 5.5
```

Comment:
Should we add a provision for users to input the number through command line?

That would enable us to test the rounding function without changing the actual file.

Reply:
Maybe if no input is provided, it rounds the specific `5.5`?